scripts/i18n/extractMessages.js

```diff
@@ -0,0 +1,93 @@
+const babel = require('@babel/core');
```

Comment:
need cr ?

Reply:
it does indeed, let me just use my handy new lint plugin to fix that...
and done